src/old-api/external-docs.ts

```diff
@@ -14,4 +15,36 @@ export class ExternalDocs extends SpecificationExtensionsModel<v2.ExternalDocume
   description() {
     return description(this);
   }
+
+  hasExtensions() {
```

Comment:
I had to do that due to circular references between `tag`, `externalDocs` and `mixins` files.